templates/pronoun_frequency.html

```diff
@@ -0,0 +1,346 @@
+
+<p>This result was then compared with the overall results for Analysis #1. No significant difference was found at the p = 0.05 level. </p>
+
+<h4>By Author Gender</h4>
+
+<p>Below shows the average portion of the total number of pronouns in a given novel that are female sorted by author gender</p>
+
+<div class="row">
+    <div class="col-md-6">
+        <p>Male author: 0.24</p>
+     <svg width="300" height="200" id="svg_4"> </svg>
+<script>
+    makePieChart([{name: 'Female', val: 24, color: '#BDE894'}, {name: 'Male', val: 56, color: '#FFD1FE'}], '#svg_4')
+    style = "width:70%; margin-left:auto; margin-right:auto;"
+</script>
+    </div>
+    <div class="col-md-6">
+        <p>Female author: 0.52</p>
+     <svg width="300" height="200" id="svg_5"> </svg>
+<script>
+    makePieChart([{name: 'Female', val: 52, color: '#BDE894'}, {name: 'Male', val: 48, color: '#FFD1FE'}], '#svg_5')
+    style = "width:70%; margin-left:auto; margin-right:auto;"
+</script>
+    </div>
+</div>
+
+
+
+<p>On average, a novel by a male author in the corpus contains 24.0% female pronouns and a novel by a female author contains 52.0% female pronouns. This difference was shown to be statistically significant at the p = 0.05 level with an independent t test.</p>
+
+<p>The results for female authors and male authors were compared with the results in Analysis #1. No significant difference was found at the p = 0.05 level. This implies that female and male authors do not use gendered pronouns any more or less frequently in the subject than they do overall.</p>
+
+<h4>By Date</h4>
+
+<ul>
+<li>To 1810: 0.34</li>
+<li>1810 to 1819: 0.37</li>
+<li>1820 to 1829: 0.22</li>
+<li>1830 to 1839: 0.21</li>
+<li>1840 to 1849: 0.28</li>
+<li>1850 to 1859: 0.34</li>
+<li>1860 to 1869: 0.37</li>
+<li>1870 to 1879: 0.40</li>
+<li>1880 to 1889: 0.34</li>
+<li>1890 to 1899: <code>NaN</code></li>
+<li>1900 on: 0.32</li>
+</ul>
+
+<h4>By Publication Location</h4>
+
+<ul>
+<li>United Kingdom: 0.33</li>
+<li>United States: 0.33</li>
+<li>Other: 0.32</li>
+</ul>
+
+<p>As in Analysis #1, no patterns or significant differences were found for date of publication or publication location.</p>
+
+<h2>Analysis #3 - Frequency of Each Type of Gendered Pronoun that are Subject Pronouns</h2>
+
+<p>Code from gender_pronoun_freq_analysis.py was used to determine the average proportion of the total number of
+pronouns of a given gender that are subject pronouns for each novel in the Gutenberg corpus. Then, results were binned by author gender, date published, and location of publication.</p>
+
+<p>This analysis is separated into two parts, 3a and 3b. 3a is the analysis of female pronouns and 3b is the analysis of male pronouns.</p>
+
+<h4>Overall</h4>
+
+<p>Below are the average proportion of the pronouns of the given gender that are subject pronouns.</p>
+
+<div class="row">
```

Comment:
@sophiazhi @meesuekim -- come nab me when you get here while working through this review, and I'll explain some bits about where to put <script> tags -- it matters for speed of page loading.